src/components/pages/callouts/CalloutIntroPanel.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <CalloutSidePanel :show="true" @close="$emit('close')">
+    <AppHeading class="mb-4">{{ $props.callout.title }}</AppHeading>
```

Comment:
You don't need `$props`, props are made globally available in the `<template>`

```vue
<AppHeading class="mb-4">{{ callout.title }}</AppHeading>
```

Could you remove all the `$props` references for consistency with the wider codebase?